### PR TITLE
Fixes the IRC bot locking up on reconnection

### PR DESCRIPTION
**Pull request by tkdrg**

### CODE REVIEW

File: bot/config.py [hunk 1]
```diff
@@ -2,8 +2,8 @@
 server = "irc.rizon.net"
 port = 6667
 
-channels = ["#asdfgbus", "#botbus"]
-defaultchannel = "#asdfgbus"
```
Comment (KazeEspada) [inline bot/config.py]: What the fuck happened here?

Comment (tkdrg) [inline bot/config.py]: sdfgsdfgasfwe

**Comment by MrStonedOne** [comment]
manually applying this to the bots.

**Comment by optimumtact** [comment]
what was causing it to hang?

File: bot/config.py (hunk 1 above)
Comment (Iamgoofball) [inline bot/config.py]: fwehdofhzf?

**Comment by tkdrg** [comment]
the irc socket on the nudge thread wasn't updating on reconnects
but apparently this doesn't work properly

**Pull request closed without merging**